src/com/tw/go/plugin/nuget/NuGetFeedDocument.java

```diff
@@ -30,7 +31,12 @@ Date getPublishedDate() {
     }
 
     private String getProperty(NodeList properties, String name) {
-        return firstOf(properties).getElementsByTagNameNS(SCHEMA_ADO_DATASERVICES, name).item(0).getTextContent();
```

Comment:
This could be:
```
Node item = firstOf(properties).getElementsByTagNameNS(SCHEMA_ADO_DATASERVICES, name).item(0);
return item == null ? "" : item.getTextContent();
```